metricbeat/module/php_fpm/proc/_meta/fields.yml

```diff
@@ -0,0 +1,74 @@
+- name: proc
+  type: group
+  description: >
+    `proc` contains the metrics that were obtained from a PHP process managed by
+    the PHP-FPM process pool.
+  fields:
+    - name: pid
+      type: keyword
+      description: >
+        The PID of the process.
+
+    - name: state
+      type: keyword
+      description: >
+        The state of the process (`Idle`, `Running`, ...).
+
+    - name: start_time
+      type: long
+      description: >
+        The date and time the process has started.
+
+    - name: start_since
+      type: long
+      description: >
+        Number of seconds since the process has started.
+
+    - name: requests
+      type: long
+      description: >
+        The number of requests the process has served.
+
+    - name: request_duration
+      type: long
+      description: >
+        The duration in microseconds of the requests.
+
+    - name: request_method
+      type: keyword
+      description: >
+        The request method (`GET`, `POST`, ...).
+
+    - name: request_uri
+      type: keyword
+      description: >
+        The request URI with the query string.
+
+    - name: content_length
+      type: long
+      description: >
+        The content length of the request (only with `POST`).
+
+    - name: user
+      type: keyword
+      description: >
+        The user (`PHP_AUTH_USER`) (or `-` if not set).
+
+    - name: script
+      type: keyword
+      description: >
+        The main script called (or `-` if not set).
+
+    - name: last_request_cpu
```

Comment:
last_request.cpu

In case it is a percent value, I would suggest `last_request.cpu.pct` and make it a `scaled_float`.